crates/supervisor/core/src/reorg/task.rs

```diff
@@ -44,6 +49,23 @@ where
             );
         })?;
 
+        debug!(
```

Comment:
```suggestion
        trace!(
```
this is business as usual, trace log is sufficient